api/definitions.yaml

```diff
@@ -380,3 +396,8 @@ Timestamp:
   type: string
   format: date-time
   description: A date-time field in ISO-8601 format
+WebUrl:
+  type: string
+  format: uri
+  description: An http(s) url.
+  pattern: "^http(s)?://[-a-zA-Z0-9@:%._+~#=]{2,256}[.][a-z]{2,6}\\\\b([-a-zA-Z0-9@:%_+.~#?&/=]*)$"
```

Comment:
we want this to be always `https`

Reply:
I've added a custom type to the italia-ts-common repo:

https://github.com/teamdigitale/italia-ts-commons/pull/10